conf/base.config

```diff
@@ -75,4 +75,8 @@ process {
     container = {(params.annotation_cache && params.vep_cache) ? 'nfcore/sarek:dev' : "nfcore/sarekvep:dev.${params.genome}"}
     errorStrategy = {task.exitStatus == 143 ? 'retry' : 'ignore'}
   }
+  withLabel:sentieon {
+    beforeScript = {params.sentieon ? 'module load sentieon/201808.05' : ''}
```

Comment:
That should be moved to the `munin` config in nf-core/configs as it's highly specific.